tests/arithmetic/bad/division_by_zero.catala_en

```diff
@@ -32,90 +32,39 @@ scope Money:
 
 
 ```catala-test-inline
-$ catala Interpret -s Dec
-[ERROR] division by zero at runtime
+$ catala test-scope Dec
```

Comment:
Could you do a PR in https://gitlab.adullact.net/dgfip/ir-catala to make sure we use `test-scope` everywhere and no longer `Interpret -s` anymore? Can you remind me the difference between the two again?

Reply:
Yes!

Basically the purpose is to be able to run the test with various combination of interpreter options (dcalc/lcalc, exceptions, etc.), so the special syntax is purposefully less explicit than the real catala command `interpret`.

See the `make testuite` target; quoting `clerk test --help`:

        --test-flags=FLAGS [...]
           Flags to pass to the catala interpreter on catala test-scope tests.
           Comma-separated list. A subset may also be applied to the
           compilation of modules, as needed. [...] NOTE: if this is set, all inline
           tests that are not `catala test-scope` are skipped to avoid redundant
           testing.